packages/apollo-gateway/src/loadServicesFromRemoteEndpoint.ts

```diff
@@ -22,31 +26,34 @@ export async function getServiceDefinitionsFromRemoteEndpoint({
   let isNew = false;
   // for each service, fetch its introspection schema
   const services: ServiceDefinition[] = (await Promise.all(
-    serviceList.map(service => {
-      if (!service.url) {
```

Comment:
Was it intentional to remove this? I don't believe removing is necessary in the scope of this PR, but please correct me if I'm wrong. Restoring this would also remove the need to cast url to `<string>` below, since we'd be guaranteeing the truthiness of `serviceDefinition.url`.

Reply:
I removed this check here because the same check exists when creating the data source. Since the url is used in this method as well it makes sense to retain the check here as well. I'll restore this check here.